### PR TITLE
Bicaws 1875 Use native types

### PR DESCRIPTION
**Pull request by C-gyorfi**
BREAKING CHANGE: Use native boolean types in the output-data JSON files
where strings are used for boolean values.
Updated:
- crest-disposal.json
- result-code.json
- offence-code.json

### CODE REVIEW

File: src/pnc-download/convertXls.test.ts [hunk 1]
```diff
@@ -5,7 +5,7 @@ const expectedCCJSOffenceCodesFile = "test-data/expected-output/pnc-ccjs-expecte
 const expectedACPOOffenceCodesFile = "test-data/expected-output/pnc-acpo-expected.json"
 
 describe("convertXls", () => {
-  it("should convert PNC CCJS file", () => {
+  it.skip("should convert PNC CCJS file", () => {
```
Comment (monotypical) [inline src/pnc-download/convertXls.test.ts:8]: would this work with the expected result changed to use native types? If not I think leaving skipped tests in the file clutters it up, they'll be in git if we need them in future

**Review by monotypical — CHANGES_REQUESTED** [review]
Does the Bichard java application need to be updated to use the new format at the same time? I think it will automatically pull in these breaking changes at the minute

File: src/pnc-download/convertXls.test.ts (hunk 1 above)
Comment (C-gyorfi) [reply src/pnc-download/convertXls.test.ts:8]: No, it is the same behaviour we just never run these tests in the CI, I'll try fixing them instead!

**Comment by C-gyorfi** [comment]
> Does the Bichard java application need to be updated to use the new format at the same time? I think it will automatically pull in these breaking changes at the minute

There is a GH workflow that opens a PR automatically in next repo, but it shouldn't merge automatically, we will need to make the changes to next repo to avoid breaking bichard

**Review by monotypical — APPROVED** [review]

File: src/pnc-download/convertXls.test.ts (hunk 1 above)
Comment (C-gyorfi) [reply src/pnc-download/convertXls.test.ts:8]: Fixed the test in this PR https://github.com/ministryofjustice/bichard7-next-data/pull/37

**Pull request merged**